indico/htdocs/js/indico/jquery/palettepicker.js

```diff
@@ -0,0 +1,112 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function($) {
+    $.widget('indico.palettepicker', {
+
+        options: {
+            availableColors: [],
+            onSelect: null,
+            numColumns: 5
+        },
+
+        _create: function() {
+            var self = this;
+            var element = this.element;
+            var paletteTable = $('<table>', {'class': 'palette-picker'});
+            var availableColors = this.options.availableColors;
+            var tr = this._createTableRow();
+
+            $.each(availableColors, function(index) {
+                var $color = this;
```

Comment:
why `$color`? it's not even a jquery object... (also, since you already take index from the arguments i'd take this one from the args too, i.e. `function(index, color)`)